pkg/setting/setting_grafana_javascript_agent.go

```diff
@@ -1,32 +1,36 @@
-	EndpointBurst                       int    `json:"-"`
-	AllInstrumentationsEnabeld          bool   `json:"allInstrumentationEnabeld"`
-	ErrorInstrumentalizationEnabled     bool   `json:"errorInstrumentalizationEnabled"`
-	ConsoleInstrumentalizationEnabled   bool   `json:"consoleInstrumentalizationEnabled"`
-	WebVitalsInstrumentalizationEnabled bool   `json:"webVitalsInstrumentalizationEnabled"`
-	TracingInstrumentalizationEnabled   bool   `json:"tracingInstrumentalizationEnabled"`
-	InternalLoggerLevel                 int    `json:"internalLoggerLevel"`
-	ApiKey                              string `json:"apiKey"`
+	EndpointRPS   int `json:"-"`
+	EndpointBurst int `json:"-"`
+
+	// Faro config
+	Enabled                               bool   `json:"enabled"`
+	CustomEndpoint                        string `json:"customEndpoint"`
+	ApiKey                                string `json:"apiKey"`
+	InternalLoggerLevel                   int    `json:"internalLoggerLevel"`
+	ConsoleInstrumentalizationEnabled     bool   `json:"consoleInstrumentalizationEnabled"`
+	PerformanceInstrumentalizationEnabled bool   `json:"performanceInstrumentalizationEnabled"`
+	CSPInstrumentalizationEnabled         bool   `json:"cspInstrumentalizationEnabled"`
+	TracingInstrumentalizationEnabled     bool   `json:"tracingInstrumentalizationEnabled"`
+	WebVitalsAttributionEnabled           bool   `json:"webVitalsAttributionEnabled"`
 }
 
 func (cfg *Cfg) readGrafanaJavascriptAgentConfig() {
 	raw := cfg.Raw.Section("log.frontend")
 	cfg.GrafanaJavascriptAgent = GrafanaJavascriptAgent{
-		Enabled:                             raw.Key("enabled").MustBool(true),
-		CustomEndpoint:                      raw.Key("custom_endpoint").MustString("/log-grafana-javascript-agent"),
-		EndpointRPS:                         raw.Key("log_endpoint_requests_per_second_limit").MustInt(3),
-		EndpointBurst:                       raw.Key("log_endpoint_burst_limit").MustInt(15),
-		AllInstrumentationsEnabeld:          raw.Key("instrumentations_all_enabled").MustBool(false),
-		ErrorInstrumentalizationEnabled:     raw.Key("instrumentations_errors_enabled").MustBool(true),
-		ConsoleInstrumentalizationEnabled:   raw.Key("instrumentations_console_enabled").MustBool(true),
-		WebVitalsInstrumentalizationEnabled: raw.Key("instrumentations_webvitals_enabled").MustBool(true),
-		TracingInstrumentalizationEnabled:   raw.Key("instrumentations_tracing_enabled").MustBool(true),
-		InternalLoggerLevel:                 raw.Key("internal_logger_level").MustInt(0),
-		ApiKey:                              raw.Key("api_key").String(),
+		EndpointRPS:   raw.Key("log_endpoint_requests_per_second_limit").MustInt(3),
+		EndpointBurst: raw.Key("log_endpoint_burst_limit").MustInt(15),
+
+		// Faro config
+		Enabled:                               raw.Key("enabled").MustBool(false),
+		CustomEndpoint:                        raw.Key("custom_endpoint").MustString("/log-grafana-javascript-agent"),
```

Comment:
Given that it has been set in the default config, can we remove it here too? Or do we set both?